src/app/components/MediaLoader/configs/tvMedia.ts

```diff
@@ -0,0 +1,49 @@
+import filterForBlockType from '#lib/utilities/blockHandlers';
+import { ConfigBuilderProps, ConfigBuilderReturnProps } from '../types';
+import buildPlaceholderConfig from '../utils/buildPlaceholderConfig';
+
+export default ({
+  blocks,
+  basePlayerConfig,
+  translations,
+}: ConfigBuilderProps): ConfigBuilderReturnProps => {
+  const { model: tvMediaBlock } = filterForBlockType(blocks, 'tvMedia');
+
+  const placeholderConfig = buildPlaceholderConfig({
+    title: tvMediaBlock.episodeTitle,
+    duration: tvMediaBlock.versions[0].duration,
```

Comment:
I would add optional chaining to these nested object accessors, particularly surrounding array -> property access as it could throw an error if the data is malformed.

Reply:
Do you mean as follows:
`title: tvMediaBlock?.episodeTitle,
  duration: tvMediaBlock?.versions[0].duration,`

Reply:
Access such as `tvMediaBlock?.episodeTitle` is less problematic as it will just return `undefined`.

For array -> property access though it could throw an error, so I would add optional chaining like:
`tvMediaBlock?.versions?.[0]?.duration`

That will ensure that if the array `[0]` is undefined/null, then attempting to access `duration` on that won't throw an exception.